keyboards/dz60/keymaps/_bonfire/README.md

```diff
@@ -0,0 +1,16 @@
+# Docs
+
+1. https://docs.qmk.fm/#/
+2. To build, run: `make dz60:_bonfire`
+
+## Map of Layers
+
+http://www.keyboard-layout-editor.com/#/gists/b19ee1c251c908d9b5ef76965d588937
+
+![Visual Keymap](visual-keymap.jpg)
```

Comment:
[We do not allow binary files in pull requests](https://docs.qmk.fm/#/hardware_keyboard_guidelines?id=imagehardware-files). Please host this file elsewhere and point this link to the new URL.

Reply:
The general suggestion is to use Imgur.